utilities/ovs-vsctl.c

```diff
@@ -458,17 +490,43 @@ Options:\n\
   -t, --timeout=SECS          wait at most SECS seconds for ovs-vswitchd\n\
   --dry-run                   do not commit changes to database\n\
   --oneline                   print exactly one line of output per command\n",
-           program_name, program_name, ctl_get_db_cmd_usage(),
-           ctl_list_db_tables_usage(), ctl_default_db());
-    table_usage();
+           ctl_default_db());
+}
+
+static void
+vlog_options_usage(void)
+{
     vlog_usage();
+
     printf("\
-  --no-syslog             equivalent to --verbose=vsctl:syslog:warn\n");
-    stream_usage("database", true, true, true);
+  --no-syslog                 equivalent to --verbose=vsctl:syslog:warn\n");
+}
+
+static void
+other_options_usage(void)
+{
     printf("\n\
 Other options:\n\
   -h, --help                  display this help message\n\
   -V, --version               display version information\n");
+}
+
+static void
+usage(void)
+{
+    ovs_usage(program_name);
```

Comment:
Order seems random. Should this be lexicographically ordered?

Reply:
I would guess that the author of the changes was trying to preserve the existing order, and that the p4 commands were inserted following the Auto Attach commands because they both have to do with BRIDGEs.

As for the reason for the existing order ... no clue.